resources/containers/confirm-modal.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Modal from "react-modal";
+import modal_styles from "./styles/confirmation.module.css";
+import { useTranslation } from "react-i18next";
+
+interface ConfirmationModalProps {
+  isOpen: boolean;
+  onRequestClose: () => void;
+  onConfirm: () => void;
+  title: string;
+  message: string;
+  confirmButtonText: string;
+}
+
+const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
```

Comment:
We have delete confirmation, and now confirmation Modal.

Shall DeleteConfirmation use ConfirmationModal? to avoid duplicated code.

Reply:
The confirm modal is the base used by all other confirm modal